### ISSUE TITLE
Order lists by GitHub number of stars

### ISSUE THREAD

**Issue by pedro-vicente** [aljazceru/awesome-nostr#410]
Order lists by GitHub number of stars, to see which one is the top rated . Ideally by programming languages too, or a matrix of features.

**Comment by aljazceru** [comment]
yeah someday i will throw this into a different format that will be better for sorting but atm i have no time

**Comment by kehiy** [comment]
its good to support sorting. but default sorting by start is the worst idea. more start doesn't mean better code.

**Comment by bitkarrot** [comment]
Sort by more recently updated, there are so many ded repos that are unmaintained, I really want to see the ones that are kept up to date.

**Comment by steve02081504** [comment]
> yeah someday i will throw this into a different format that will be better for sorting but atm i have no time

maybe write an script and auto update by CI?

**Comment by steve02081504** [comment]
Opened PR #724 which implements this: machine-readable JSON data + weekly CI sorting with configurable strategies (default hybrid: >=500-star entries ranked by stars, the rest by last activity — so active small projects get promoted and popular-but-dormant ones don't sink).
